caffe2/onnx/backend.cc

```diff
@@ -1361,6 +1362,17 @@ Caffe2Ops Caffe2Backend::CreateLRN(
   return c2_op;
 }
 
+Caffe2Ops Caffe2Backend::CreateInt8Reshape(
```

Comment:
Can we share the code between CreateReshape and CreateInt8Reshape instead of copying it?